ember-phone-input/src/components/phone-input.ts

```diff
@@ -29,10 +43,70 @@ import '../styles/styles.css';
   @public
 */
 
-export default class PhoneInputComponent extends Component {
-  @service phoneInput;
+export interface PhoneInputArgs {
+  country: string;
+  number: string | null;
+  disabled: boolean;
+  required: boolean;
+  autocomplete: string | null;
+  allowDropdown: boolean;
+  autoPlaceholder: 'aggressive' | 'off' | 'polite' | undefined;
+  customPlaceholder: string | null;
+  initialCountry: string;
+  onlyCountries: string[];
+  preferredCountries: string[];
+  separateDialCode: boolean;
+  update: (number: string | null, meta: MetaData) => void;
+  onError: (error: unknown) => void;
+}
+
+export interface PhoneInputSignature {
+  Element: HTMLInputElement;
+  Args: PhoneInputArgs;
+}
+
+export default class PhoneInputComponent extends Component<PhoneInputSignature> {
+  @service declare phoneInput: PhoneInputService;
 
+  isLoadingIti = false;
   type = 'tel';
+  _iti: intlTelInput.Plugin | null = null;
```

Comment:
Here too I feel that something like `intlTelInputInstance` would be more readable. 

question: why are we using the underscore `_` as prefix?

Reply:
Makes sense to rename it 👍
`_` prefix was used to mark private class attributes and methods. I've renamed them and added private